include/beast/http/parser_v1.hpp

```diff
@@ -150,7 +152,7 @@ class parser_v1
 
     /** Returns the parsed message.
 
-        Only valid if `complete()` would return `true`.
+        Only valid if @ref complete would return `true`.
```

Comment:
Yeah, that's because `complete` is implemented in the base class `basic_parser_v1`. If anything, this is a bug or missing feature in the xsl transform. The information is there, in the Doxygen XML output, we just need to figure out a way to format it and insert into the docs something like "inherited from base class ____".